docs/roadmap.md

```diff
@@ -1,28 +1,36 @@
 # Roadmap
-
 This tool is just one piece of a bigger journey we’re on to streamline the journey to upgrade your apps from .NET Framework to .NET 5 and beyond.
+This list is not an exhaustive list of all features and support coming – but rather the ones most requested and/or prioritized to keep your upgrades on track. If there’s anything missing from this list that you expect to see or you’re having difficulty with in your upgrade, please let us know via an [issues](https://github.com/dotnet/upgrade-assistant/issues).
 
-Below is a preview of some of the areas we want to focus on in that journey and also areas we want to invest in for this tool. If there’s anything missing from this proposed list that you expected to see or you’re having difficulty with in your upgrade, please let us know via an [issue]( https://github.com/dotnet/upgrade-assistant/issues).
-
-This list is not meant to be exhaustive of all features and [issues](https://github.com/dotnet/upgrade-assistant/issues) that could be supported, just a high level overview.
+The priority or state for an issue can change at any time.
 
 **1. Pre-migration analysis**
 
-Example areas:
-- Up front checks for technologies that might not be supported in .NET 5+ or could make the upgrade effort large in cost (links to docs, options available, best practices, etc.).
-- More visibility into NuGet dependencies (e.g. newer versions, do they support .NET 5+?)
-- Recommendations for TFM (target frameworks) to move to
+Take a look at the design doc for [design details](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md) and phases of implementation.
+
+- **Committed**
+  - [Loose assemblies analysis](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md#loose-assembly-analysis)
+- **On Deck**
+  -	More visibility into [NuGet dependencies](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md#nuget-package-dependecy-analysis) (e.g. newer versions, do they support .NET 5+?), transitive dependencies, and [inter-project dependencies](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md#inter-project-dependencies)
+  -	[Up front checks for technologies that might not be supported in .NET 5+](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md#surface-unsupported-api-categories) or could make the upgrade effort large in cost (links to docs, options available, best practices, etc.).
+  -	Recommendations for TFM (target frameworks) to move to
```

Comment:
This is most likely axed, so remove it from the list for now.

Reply:
Removed.